### PR TITLE
Controller draft, validator, converter

### PR DESCRIPTION
**Pull request by GabrielRendon**
first controller draft
common validator
common converter

### CODE REVIEW

File: src/com/jalasoft/search/MVCSearch.java [hunk 1]
```diff
@@ -0,0 +1,17 @@
+package com.jalasoft.search;
+import com.jalasoft.search.model.SearchModel;
+
```
Comment (miguelcg88) [inline src/com/jalasoft/search/MVCSearch.java]: Please add class description

File: src/com/jalasoft/search/common/Convertor.java [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+/**
+ * Convertor
+ *
+ * @version 1.00 23 Mar 2018
+ * @author Gabriel Rendon  */
+public class Convertor {
+    /**
+     MEGABYTE divides bytes (2 times 2014)*/
+    private static final long MEGABYTE = 1024L * 1024L;
+
```
Comment (miguelcg88) [inline src/com/jalasoft/search/common/Convertor.java]: I think method names should start with lowercase

File: src/com/jalasoft/search/common/Validator.java [hunk 3]
```diff
@@ -0,0 +1,103 @@
...
+ * Copyright (c) Jalasoft
+ *
+ * This software is the confidential and proprietary information of Jalasoft
+ * ("Confidential Information").  You shall not
+ * disclose such Confidential Information and shall use it only in
+ * accordance with the terms of the license agreement you entered into
+ * with Jalasoft.
+ */
+package com.jalasoft.search.common;
+import java.io.File;
```
Comment (miguelcg88) [inline src/com/jalasoft/search/common/Validator.java]: I think we need an space between imports and package

File: src/com/jalasoft/search/MVCSearch.java (hunk 1 above)
Comment (GabrielRendon) [reply src/com/jalasoft/search/MVCSearch.java]: Done

**Review by miguelcg88 — CHANGES_REQUESTED** [review]
added comments

File: src/com/jalasoft/search/common/Validator.java (hunk 3 above)
Comment (GabrielRendon) [reply src/com/jalasoft/search/common/Validator.java]: done

**Review by miguelcg88 — APPROVED** [review]

**Pull request merged**